source/components/resources/rsaddr.c

```diff
@@ -441,7 +441,8 @@ AcpiRsGetAddressCommon (
     /* Validate the Resource Type */
 
     if ((Address.ResourceType > 2) &&
-        (Address.ResourceType < 0xC0))
+        (Address.ResourceType < 0xC0) &&
+        (address.resource_type != 0xA))
```

Comment:
You might have left this line (445) from your Linux implementation! This needs to be corrected to (Address.ResourceType != 0xA) ) just like the couple of lines above that inside the same 'if' statement predicate. The hex value might look even better if it is 0x0A to maintain the consistency in the same usage. Please fix this and then it will be all good I think, thanks.

Reply:
Thank you.  You are correct, the change was from Linux, and I missed the capitalization and naming convention differences.  I believe I have them correct in the updated version.